asking-good-questions/asking-good-questions.md

```diff
@@ -19,7 +19,7 @@ About 2 hours
 - Write (Check for Understanding)
 
 ### Motivation
-The ability to ask questions that lead to useful answers is a must-have skill for software engineers because they are constantly solving problems. Asking the right questions helps engineers arrive at solutions more quickly and make faster progress.
+The ability to ask questions that lead to useful answers is a must-have skill for software engineers because they are constantly solving problems. Asking the right questions helps engineers arrive at solutions more quickly and make faster progress. The way a person asks questions deflects their personality.
```

Comment:
I don’t think this extra line is necessary. What we really need is an example. Find a quote about a company liking engineers who ask good questions or who are naturally curious.